app/controllers/api/v1/visualization/explore_controller.rb

```diff
@@ -90,7 +90,8 @@ def show
             # the react refactor is no longer feature-flagged
             spatialGroupNames: spatial_group_options.map { |opt| opt[:name] },
             spatialGroups: spatial_group_options,
-            clusterPointAlpha: @study.default_cluster_point_alpha
+            clusterPointAlpha: @study.default_cluster_point_alpha,
+            defaultColorProfile: @study.default_color_profile
```

Comment:
For consistency, this should be `colorProfile` or `clusterPointAlpha` should be `defaultClusterPointAlpha`.

```suggestion
            colorProfile: @study.default_color_profile
```